recipes/imread/meta.yaml

```diff
@@ -11,21 +10,19 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    - add_sys_prefix_include.patch
 
 build:
   number: 0
-  # Patches for Windows compilation are welcome, currently fails to link
-  skip: true  # [win]
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script:
+    - cp %LIBRARY_PREFIX\libpng.lib  %LIBRARY_PREFIX\png.lib  # [win]
```

Comment:
Sorry, I asked that question poorly. If `libpng.lib` is renamed to `png.lib`, does it look for `png.dll` or does the `.lib` file specify to look for `libpng.dll` internally somehow?